docs/fastdds/env_vars/env_vars.rst

```diff
@@ -252,6 +252,40 @@ The following example shows how to set the address of two remote discovery serve
     (that has been initialized with this environment variable previously) if loaded from an environment file using
     :ref:`env_vars_fastdds_environment_file`.
 
+.. _env_vars_easy_mode:
+
+``EASY_MODE`` (Beta)
+--------------------
+
+.. warning::
+    Discovery Server ``EASY_MODE`` is a Beta feature. Its functionalities are still under development and they might contain bugs.
+    Its final version will be available soon. Any feedback is appreciated and can be provided at:
+
+Setting ``EASY_MODE`` to an IP value allows a participant to automatically enter the `Discovery Server EASY_MODE<https://docs.vulcanexus.org/en/latest/rst/enhancements/ds_auto_discovery/ds_auto_discovery.html>`__.
+This mode completely disables **multicast communication**, and relies on Discovery Servers for discovery purposes.
+
+In ``EASY_MODE`` a new Discovery Server will be automatically spawned locally in the given domain, pointing to another Discovery Server located in the specified IP.
```

Comment:
It may be helpful to include a link to the DDS domain
```suggestion
In ``EASY_MODE`` a new Discovery Server will be automatically spawned locally in the given [domain](https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/domain/domain.html#domain), pointing to another Discovery Server located in the specified IP.
```